README.md

```diff
@@ -200,15 +201,28 @@ await timber.remove();
     
         `npm install oracledb --save`
     
-    Install only *one* of them, depending on which database you use.
-    
-    To make the Oracle driver work, you need to follow the installation instructions from 
-    [their](https://github.com/oracle/node-oracledb) site.
+        Install only *one* of them, depending on which database you use.
+
+        To make the Oracle driver work, you need to follow the installation instructions from
+        [their](https://github.com/oracle/node-oracledb) site.
 
     * for **MongoDB** (experimental)
 
         `npm install mongodb --save`
 
+    * for **NativeScript**

```

Comment:
As no other mobile driver is defined in this list I would like to move the explanation to docs/supported-platforms.md